apps/sms/js/thread_ui.js

```diff
@@ -1864,7 +1860,9 @@ var ThreadUI = {
     }
 
     // Removing from DOM all messages to delete
+    var messageMap = Threads.active.messages;
     for (var i = 0, l = list.length; i < l; i++) {
+      messageMap.delete(+list[i]);
```

Comment:
we should implement a Threads.unregisterMessage instead, to achieve good encapsulation.

(I swear I saw some code that already implemented this, but it's likely in some old uncommited PR :/)